app/models/mixins/cinder_manager_mixin.rb

```diff
@@ -18,24 +18,10 @@ module CinderManagerMixin
     private
 
     def ensure_cinder_managers
-      created = ensure_cinder_manager
+      ensure_cinder_manager
```

Comment:
@kbrock this looks like a pointless method now if it just calls `ensure_cinder_manager` right?